SunEngine.Core/Presenters/SearchPresenter.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using LinqToDB;
+using SunEngine.Core.DataBase;
+using SunEngine.Core.Services;
+
+namespace SunEngine.Core.Presenters
+{
+    public interface ISearchPresenter
+    {
+        Task<UserSearchInfoView[]> SearchByUserNameAndLink(string searchString);
+    }
+    
+    public class SearchPresenter : DbService, ISearchPresenter
+    {
+        public SearchPresenter(DataBaseConnection db) : base(db)
+        {
+        }
+
+        public Task<UserSearchInfoView[]>  SearchByUserNameAndLink(string searchString)
```

Comment:
The name of the method doesn`t correspond to its actions. (NameAndLink, but in code NameOrLink)